configure.ac

```diff
@@ -2929,7 +2929,7 @@ AC_DEFINE([STDC_HEADERS], [1],
 
 # checks for header files
 AC_CHECK_HEADERS([ \
-  alloca.h asm/types.h bluetooth.h conio.h direct.h dlfcn.h endian.h errno.h fcntl.h grp.h \
+  alloca.h asm/types.h bluetooth.h conio.h direct.h dlfcn.h endian.h errno.h execinfo.h fcntl.h grp.h \
```

Comment:
Hm, it appears a couple systems (freebsd, android) has `execinfo.h` but doesn't have `backtrace` and `backtrace_symbols`, so we'll need `AC_CHECK_FUNCS` as well.

and/or check libexecinfo: https://github.com/tytso/e2fsprogs/issues/104

Reply:
nice, this should work now - but should perhaps try `libexecinfo` as well to see if that works for bsd/etc

Reply:
For now, I want to stick with a minimal implementation. We can add additional compatibility (Windows, BSD, Android) in follow up PRs.